name/src/commonMain/kotlin/lang/temper/name/ModuleLocation.kt

```diff
@@ -321,6 +321,13 @@ data class DashedIdentifier(
          */
         val temperCoreLibraryIdentifier = DashedIdentifier("temper-core")
 
+        /**
+         * The current version number for the core library.
+         * TODO Move this and [temperCoreLibraryIdentifier] elsewhere?
+         * TODO Script to update all temper-core backends and also std with this?
+         */
+        const val temperCoreLibraryVersion = "0.6.0"
```

Comment:
We had no handy access to this, and I didn't feel like parsing project metadata files again for such. Note the TODO comments above. This can definitely be improved in the future.

Reply:
> Also TODO: Could this be a SemVer? Versions tend to get compared, so storing values in a form that has semantically useful comparison leads to fewer downstream bugs.

Good idea. #266